dbmanager.py

```diff
@@ -31,23 +40,38 @@ def exec_sql(connection, sql):
     :returns None if the connection or sql was None,
              a cursor containing the results of the statement otherwise
     :raises A MySqlError (likely due to an error in the SQL statement syntax)
+    
+    example: 
+        
+        from testdb.py
+        import dbmanager
```

Comment:
Maybe I'm just misunderstanding, but what's going on with these two lines? Shouldn't it be "from X import Y" in a single line?